OnlineShopUI/online-shop-ui/src/App.js

```diff
@@ -1,23 +1,18 @@
-import logo from './logo.svg';
-import './App.css';
-
+import './App.scss';
+import BlockProduct from './components/BlockProduct';
+import Header from './components/Header';
+import Recepies from './components/Recepies';
+import Search from './components/Search';
+// import MenuBoard from './components/Menu';
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Header />
+      <Search />
+      {/* <MenuBoard /> */}
+      <Recepies />
+      <BlockProduct />
+      <BlockProduct />
```

Comment:
Can we count it as code duplicating, for instance if we need to add 10 BlockProduct components?